paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -24,14 +24,16 @@ internal class PaymentOptionsViewModel(
     eventReporter: EventReporter,
     customerRepository: CustomerRepository,
     workContext: CoroutineContext,
-    application: Application
+    application: Application,
+    logger: Logger
 ) : BaseSheetViewModel<PaymentOptionsViewModel.TransitionTarget>(
     config = args.config,
     prefsRepository = prefsRepository,
     eventReporter = eventReporter,
     customerRepository = customerRepository,
     workContext = workContext,
-    application = application
+    application = application,
+    logger = logger
```

Comment:
seems not used?

Reply:
@ccen-stripe I had the wrong base for this branch.  This comment might be more appropriate for this review: https://github.com/stripe/stripe-android/pull/4191.

I am not exactly sure what was appearing as unused here, was it the logger parameter or the entire constructor?